src/os/ui/ex/addexportoptions.js

```diff
@@ -0,0 +1,86 @@
+goog.provide('os.ui.addExportCtrl');
+goog.provide('os.ui.addExportDirective');
```

Comment:
New angular directives should be made in the new goog.module() format. You can just...
1. Run the es6-transform on this file,
2. Delete the shim that it makes, 
3. Rename the file to "addexportoptionsui.js" and the module to "AddExportOptionsUI", (to follow the new-ish naming convention), and
4. Make a small tweak to the goog.require's elsewhere that pull in the directive